CONTRIBUTING.md

```diff
@@ -31,7 +31,9 @@ is [purposefully excluded from the project's `Gemfile`][exclude].
 [foreman]: https://github.com/ddollar/foreman
 [exclude]: https://github.com/ddollar/foreman/pull/437#issuecomment-41110407
 
-The application will run locally at `http://localhost:5000/`
+The application will run locally at `http://localhost:5000/`. 
+
+If you have previously run a project on a different port, a `.foreman` file may be generated at the root of your directory. If so, make sure that this file is set to port `5000` or you will be unable to authenticate locally with MyUSA.
```

Comment:
you OK with doing line breaks at around 80 chars? makes it a lot easier to leave inline comments in github and navigate code/words when working on a project